README.md

```diff
@@ -1,2 +1,89 @@
 # github-release-orb
-Civis Analytics' CirleCI Orb repo.
+Civis Analytics' CircleCI Orb repo.
+
+## Usage
+
+Example Config:
+
+```
+version: 2.1
+
+orbs:
+  github-release: civisanalytics/github-release@x.y.z
+
+jobs:
+  release:
+    executor: github-release/github
+  steps:
+    - github-release/<command>
+```
+
+## Build
+In order to build the orb you can run the following command in the root directory of this repository.
+
+`circleci config pack src > orb.yml`
+
+Do not commit the created `orb.yml` file. This file is for publishing to CircleCI when finished.
```

Comment:
Should there be instructions on how to publish new versions of the orb? Or a link to docs on how to publish, if there are circle maintained docs

Reply:
I can add those